src/types/hash.zig

```diff
@@ -0,0 +1,15 @@
+const std = @import("std");
+
+const Self = @This();
+
+bytes: [32]u8,
+
+/// Create a zero hash
+pub fn zero() Self {
```

Comment:
why ?

Reply:
rather `newZeroed`, we are talking about an array, not a single number. So if it was u8.zero, it would be nice, but it is not obvious what zero means for an array. newZeroed is obvious